ruvnl_consumer_app/app.py

```diff
@@ -8,22 +8,125 @@
 import sys
 
 import click
-from pvsite_datamodel import DatabaseConnection
-
+import pandas as pd
+import requests
+from pvsite_datamodel import DatabaseConnection, SiteSQL
+from pvsite_datamodel.read import get_sites_by_country
+from pvsite_datamodel.write import insert_generation_values
+from sqlalchemy.orm import Session
 
 log = logging.getLogger(__name__)
 
+DATA_URL = "http://sldc.rajasthan.gov.in/rrvpnl/read-sftp?type=overview"
```

Comment:
could we do this as `data_url = os.enviorn('DATA_URL', "http://sldc.rajasthan.gov.in/rrvpnl/read-sftp?type=overview")`, then if it changes, it really easy to do that